source/manual/govuk-site-search.html.md

```diff
@@ -0,0 +1,111 @@
+---
+owner_slack: "#govuk-search"
+title: "GOV.UK Site search: How it works"
+parent: "/manual.html"
+layout: manual_layout
+section: Search
+type: learn
+---
+
+## Overview
+
+The core functionality of the site search stack consists of three main parts:
+
+- **Synchronisation** (sometimes called “indexing”): Continuous ingestion of data from the GOV.UK content ecosystem (via Publishing API), extracting relevant content and metadata, and storing it in Google Vertex AI Search
+- **Querying**: Providing an API that acts as a translation layer between Finder Frontend and Google Vertex AI Search, executing search queries with optional filters, and transforming results into the expected data format for display to the user
+- **Analytics pipeline**: Daily ingestion of user interaction data from GA4 used by Vertex AI Search to improve the ranking model.
+
+### Explainer: Google product names
+
+We use Vertex AI Search (VAIS, “Vertex” for short) to describe the search product we use.
+
+During the lifetime of our implementation project, this search product has been renamed several times (starting off as “Gen AI App Builder: Enterprise Search”, then “Vertex AI Search and Conversation”, now just “Vertex AI Search”) and now comes under the umbrella of (Vertex AI) Agent Builder as part of Google’s overall Vertex AI brand for all things AI. It’s not to be confused with the plain Vertex AI product though (their overall AI development platform which provides access to Gemini LLMs).
+
+Discovery Engine is the name of the API used by integrations with VAIS, which is partly shared with a similar-but-different product named Cloud Retail Search (CRS). In the code and code-adjacent documentation, we prefer to use Discovery Engine naming over VAIS where possible to future proof ourselves for the next product renaming.
+
+## Architecture
+
+### Architecture summary
+
+![site-search-diagram][link-1]
+
+Broadly, the new search stack for site search introduces the following new components to the GOV.UK technical ecosystem:
+
+- A deployment of **Vertex AI Search** (VAIS) in Google Cloud Platform (GCP) managed by Terraform, comprising per-environment GCP projects with a datastore (and schema), an engine, and serving configuration and controls
+
+- **Search API v2**: A Ruby on Rails application deployed on the standard GOV.UK AWS Kubernetes infrastructure, providing an API for querying (search and autocomplete) the VAIS serving configuration, a synchronisation module for updating the state of the VAIS datastore, and a scheduled task performing ongoing automated relevance evaluations.
```

Comment:
This makes me think it might be worth having an explainer somewhere of the difference between search v1 vs v2 (maybe it exists but Might be useful to have it somewhere in these docs?)

Reply:
Do you mean how they work? That might take a bit of time as it's been a while since anyone dug into the ins and outs of how search-api v1 works,
or just which bits are powered by search-api / Elasticsearch and which bits search-api-v2 / VAIS?

Reply:
@beccapearce There's this existing page: https://docs.publishing.service.gov.uk/manual/govuk-search.html
All of these pages should appear alongside this existing page under a new "Search" section in the manual.

Reply:
@leenagupte yep I think that page does it!!  I'll resolve this :)